src/main.browser.ts

```diff
@@ -32,6 +33,6 @@ export const initializeBKTClient = async (
   user: BKTUser,
   timeoutMillis = 5_000,
 ): Promise<void> => {
-  const component = createBrowserComponent(config, user)
+  const component = createBrowserComponent(config, toUser(user))
```

Comment:
Because the BKTUser and User have different field names for the user attributes, we must set the User when creating the component. Otherwise, the data field on the server will always be empty.